routes/welcomeRouter.js

```diff
@@ -0,0 +1,7 @@
+import {Router} from 'express'; 
```

Comment:
Are you sure about this import? Does`Router` exist inside `express`?
COnsider using
```
import express from 'express';
var router = express.Router();
```

Reply:
Ok, let me use this `import express from 'express';`

Reply:
Is better to use this 
`import express from 'express';
const router = express.Router();`
That is correct, thank you.